sdk-setup/src/sdk-init

```diff
@@ -99,6 +99,30 @@ bad_usage()
 
 inside_build_engine() [[ -f /etc/mer-sdk-vbox ]]
 
+with_tmp_file()
+{
+    local file=$1 cmd=("${@:2}")
```

Comment:
maybe $output_file instead of $file?

Reply:
I kind of agree with the suggestion but OTOH this is a copy of a snippet that exists in multiple files already, so I would prefer keeping all occurrences in sync and only touch it if necessary :)

Reply:
I didn't know that. I guess that makes sense.